ui/src/app/richskill/detail/rich-skill-manage/action-bar/action-bar-vertical/manage-skill-action-bar-vertical.component.spec.ts

```diff
@@ -21,13 +21,15 @@ import any = jasmine.any
     <app-manage-skill-action-bar-vertical
       [skillUuid]="mySkillUuid"
       [skillName]="mySkillName"
+      [skillPublicUrl]="mySkillPublicUrl"
       [archived]="myArchived"
       [published]="myPublished">
     </app-manage-skill-action-bar-vertical>`
 })
 class TestHostComponent {
   mySkillUuid = "1234"
   mySkillName = "my skill name"
+  mySkillPublicUrl = "https://localhost:8080/skill/1234"
```

Comment:
To me, this conveys that there is secret sauce in the URL (protocol, port, path). Would it work just as will with "mockUrl" as the string value?

Reply:
sure